packages/io/stl-deserializer/README.md

```diff
@@ -25,25 +24,22 @@ npm install @jscad/stl-deserializer
 
 ## Usage
 
-
 ```javascript
-const stlDeSerializer = require('@jscad/stl-deserializer')
+const stlDeserializer = require('@jscad/stl-deserializer')
 
 const rawData = fs.readFileSync('PATH/TO/file.stl')
-const csgData = stlDeSerializer.deserialize(rawData, undefined, {output: 'csg'})
+const geometry = stlDeserializer.deserialize(rawData, 'file.stl', {output: 'csg'})
```

Comment:
output: 'geometry' :)
do we keep the file name parameter around ? perhaps we can ditch that in V2 ? (I only vaguely recall the details tbh)

Reply:
Not sure about the file name parameter. I think it only adds to the comments of the JSCAD script.